src/readers/utils.cpp

```diff
@@ -0,0 +1,78 @@
+#include "./utils.h"
+
+namespace morphio {
+#if defined(_WIN32) || defined(__WIN32__) || defined(WIN32)
+#define freelocale _free_locale
+#define strtol_l _strtol_l
+
+#ifdef MORPHIO_USE_DOUBLE
+#define strto_float _strtod_l
+#else
+#define strto_float _strtof_l
+#endif
+
+#else  // not WIN32
+
+#ifdef MORPHIO_USE_DOUBLE
+#define strto_float strtod_l
+#else
+#define strto_float strtof_l
+#endif
+
+#endif
+
+StringToNumber::StringToNumber()
+#if defined(_WIN32) || defined(__WIN32__) || defined(WIN32)
+    : locale(_create_locale(LC_ALL, "C")) {
+}
+#else
+    : locale(newlocale(LC_NUMERIC_MASK, "POSIX", nullptr)) {
+}
+#endif
```

Comment:
Same here. It's not clear to me why these are configured this way.

Reply:
done